pyverbs/providers/hns/hnsdv_enums.pxd

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: (GPL-2.0 OR Linux-OpenIB)
+# Copyright (c) 2021 HiSilicon Limited. All rights reserved.
+
+#cython: language_level=3
+
```

Comment:
Please remove all the enum values, they're not needed in Cython, as it takes the values as defined in the original .h file anyway.